tests/translator/output/all_policy_templates.json

```diff
@@ -1362,7 +1384,7 @@
                 }
               ]
             },
-            "PolicyName": "KitchenSinkFunctionRolePolicy51"
+            "PolicyName": "KitchenSinkFunctionRolePolicy52"
```

Comment:
Why are these changing?

Reply:
oh I see. I've added a policy to the test, which is now number 51, so all other policy numbers change. It's not a part of my PR

Reply:
fine